aspnetcore-features/CustomAttributes/TestProject/PrintAttributesUnitTest.cs

```diff
@@ -0,0 +1,43 @@
+using CustomAttributes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TestProject
+{
+    [TestClass]
+    public class PrintAttributesUnitTest
+    {
+        [TestMethod]
+        public void WhenCorrectInput_ThenCorrectSingleAttOutput()
+        {
+            bool result = true;
+
+            try
+            {
+                CustomAttributeHelper.GetTaskAttribute(typeof(TaskGroup));
+            }
+            catch
+            {
+                result = false;
+            }
+
+            Assert.IsTrue(result == true);
+        }
+
+        [TestMethod]
+        public void WhenCorrectInput_ThenCorrectMultipleAttOutput()
+        {
+            bool result = true;
+
+            try
+            {
+                CustomAttributeHelper.GetTaskAttributesOfMethod(typeof(TaskGroup));
+            }
+            catch
+            {
+                result = false;
+            }
+
+            Assert.IsTrue(result == true);
+        }
```

Comment:
Same thing as the previous method. Moreover, this methods barely test anything except that it works. If you have problems testing these methods, you should change the implementation so that they return something like the TaskDescriptor in the GetTaskAttribute method and the MethodInfo[] in the second one. That way you'll be able to perform some meaningful Asserts instead of these ones. If you do this, please modify the methods in the article as well.